raiden/tests/integration/api/fixtures.py

```diff
@@ -11,7 +13,13 @@
 #       the server is no longer running even though the teardown has not
 #       been invoked.
 @pytest.fixture
-def api_server_test_instance(raiden_network: List[RaidenService]) -> APIServer:
+def api_server_test_instance(raiden_network: List[RaidenService]) -> Generator:
     api_server = prepare_api_server(raiden_network[0])
 
-    return api_server
+    yield api_server
```

Comment:
I just wanted to do it in 2 steps, first checking if it's ok to use a generator instead of a function (so far so good) and later on try to make this session scoped.

If I'm not mistaken, flask documentation always use fixtures as generators and not as functions, so it could be a good idea to conform with the official documentation.

Reply:
I could only find https://flask.palletsprojects.com/en/1.1.x/testing/#the-testing-skeleton, which is using yield because there is teardown code to be executed after the test/yield. We're also losing the type information on the return value. But this is getting nit-picky. I'll also accept the code as it is above.